recipes/ctapipe/meta.yaml

```
@@ -0,0 +1,84 @@
+{% set name = "ctapipe" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 35ac6fb9db7f89879e0e1807765fb31e3e165f7af9b0adfff27dea777a6136f3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - "ctapipe-info = ctapipe.tools.info:main"
+    - "ctapipe-camdemo = ctapipe.tools.camdemo:main"
+    - "ctapipe-dump-triggers = ctapipe.tools.dump_triggers:main"
+    - "ctapipe-chargeres-extract = ctapipe.tools.extract_charge_resolution:main"
+    - "ctapipe-chargeres-plot = ctapipe.tools.plot_charge_resolution:main"
+    - "ctapipe-dump-instrument=ctapipe.tools.dump_instrument:main"
+    - "ctapipe-event-viewer = ctapipe.tools.bokeh.file_viewer:main"
+    - "ctapipe-display-tel-events = ctapipe.tools.display_events_single_tel:main"
+    - "ctapipe-display-imagesums = ctapipe.tools.display_summed_images:main"
+    - "ctapipe-reconstruct-muons = ctapipe.tools.muon_reconstruction:main"
+    - "ctapipe-display-integration = ctapipe.tools.display_integrator:main"
+    - "ctapipe-display-dl1 = ctapipe.tools.display_dl1:main"
+    - "ctapipe-stage1-process = ctapipe.tools.stage1:main"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - setuptools
+    - astropy >=4,<5
+    - bokeh >=1.0,<2.0.0a0
+    - eventio >=1.1.1,<2.0.0a0
+    - iminuit >=1.3
+    - joblib
+    - matplotlib >=3.0,<4.0.0a0
+    - numba >=0.43
+    - numpy >=1.17
+    - pandas >=0.24.0
+    - psutil
+    - scikit-learn
+    - scipy >=1.2
+    - pytables >=3.4
+    - tqdm >=4.32
+    - traitlets >=5.0.5,<6.0.0a0
+    - zstandard
+    - h5py
+
+test:
+  imports:
+    - ctapipe
+    - ctapipe.core
+    - ctapipe.image
+    - ctapipe.calib
+    - ctapipe.instrument
+    - ctapipe.visualization
+    - ctapipe.io
+  commands:
+    - ctapipe-info --version
```

Can we try to add ``pip check`` here please?

```suggestion
    - ctapipe-info --version
    - pip check
  requires:
    - pip
```